source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -68,11 +70,16 @@ def getCommentInfoFromPosition(position):
 		UIAElement=UIAElementArray.getElement(index)
 		UIAElement=UIAElement.buildUpdatedCache(UIAHandler.handler.baseCacheRequest)
 		obj=UIA(UIAElement=UIAElement)
-		if not obj.parent or obj.parent.name!='Comment':
+		if (obj.parent.role !=controlTypes.ROLE_GROUPING and not

```

Comment:
Why did you remove the ``if not obj.parent` here? This breaks if the object has no parent.

Reply:
I focused on replacing obj.name and missed it. Thanks for pointing this.